java/code/src/com/suse/manager/webui/controllers/DownloadController.java

```diff
@@ -533,6 +533,14 @@ private String getTokenForDebian(Request request) {
         return null;
     }
 
+    /**
+     * @param token the token
+     * @return the last characters of the token
+     */
+    private static String sanitizeToken(String token) {
```

Comment:
nitpick: `truncateToken` or `obfuscateToken` might be better names for this method.

Reply:
Oh I just realized that this is a port and the original PR is already merged, so nevermind this comment :)